examples/complete/elasticsearch.tf

```
@@ -0,0 +1,59 @@
+module "tamr-es-cluster" {
+  source = "git::git@github.com:Datatamer/terraform-aws-es?ref=2.1.0"
+
+  # Names
+  domain_name = "${var.name_prefix}-es"
+  sg_name     = "${var.name_prefix}-es-security-group"
+
+  # Only needed once per account, set to true if first time running in account
+  create_new_service_role = false
+
+  # In-transit encryption options
+  node_to_node_encryption_enabled = true
+  enforce_https                   = true
+
+  # Networking
+  vpc_id             = module.vpc.vpc_id
+  subnet_ids         = [data.aws_subnet.data_subnet_es.id]
+  security_group_ids = module.aws-sg-es.security_group_ids
+  # CIDR blocks to allow ingress from (i.e. VPN)
+  ingress_cidr_blocks = var.ingress_cidr_blocks
+  aws_region          = data.aws_region.current.name
+}
+
+data "aws_region" "current" {}
+
+# Security Groups
+module "sg-ports-es" {
+  source = "git::git@github.com:Datatamer/terraform-aws-es.git//modules/es-ports?ref=2.1.0"
+}
+
+module "aws-sg-es" {
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = module.vpc.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = concat(module.aws-sg-vm.security_group_ids, module.aws-emr-sg-master.security_group_ids, module.aws-emr-sg-core.security_group_ids)
```

We could create fewer rules if we use the managed security group id here instead:
```suggestion
  ingress_security_groups = concat(module.aws-sg-vm.security_group_ids, module.emr.emr_managed_sg_id
```